tests/testthat/test-generic.R

```diff
@@ -7,6 +7,12 @@ test_that("new_generic checks its inputs", {
   })
 })
 
+test_that("new_generic finds R7_dispatch calls", {
+  expect_error(new_generic("foo", "x", function(x) { }))
```

Comment:
I think you can remove this since it's tested above.

Reply:
Can do - I like the specificity here, but don't mind removing it.